streamline_training.py

```diff
@@ -16,16 +16,68 @@
 import os
 import socket
 import subprocess
+import json
+
+from typing import Dict, Text
 
 """Train LaserTagger model and export to GCP bucket."""
 
 GCP_BUCKET = "gs://trained_models_yechen/"
 
 
+def write_lasertagger_config(output_dir: Text, bert_type: Text, t2t: bool, number_of_layer: int, hidden_size: int, attention_heads: int, filter_size: int, full_attention: bool):
+    """ Write the LaserTagger configuration as a json file.
+    
+    Args:
+        file_dir: the directory where the json file will be stored
+        bert_type: the type of Bert. There are two types: Base and POS
+        t2t: If True, will use autoregressive decoder. If False, will use feedforward decoder.
+        number_of_layer: number of hidden layers in the decoder.
+        hidden_size: the size of hidden layer in the decoder.
+        attention_heads: number of attention heads in the decoder. 
+        filter_size: the size of the decoder filter.
+        full_attention: whether to use full attention in the decoder. 
+    """
+    
+    if bert_type == "Base":
+        vocab_type = 2
+        vocab_size = 28996
+    elif bert_type == "POS":
+        vocab_type = 42
```

Comment:
where are these numbers 2, 42 from?

Reply:
2 and 42 are the vocab type for segment id embedding. So for regular Bert, it is just 0 and 1 and thus 2. For the Bert trained with POS, there are 42 different POS tags, and thus the type is 42.

Reply:
Got it, you should add comment to explain these numbers or the reader will be confused.
Also it is better to give a pointer to the source of these tags.